### PR TITLE
Added website-header

### PR DESCRIPTION
**Pull request by Naufil-Shemle**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (Rehmatulla-Khan) [inline index.html:7]: nice

File: index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  </head>
+  <body></body>
```
Comment (faran4engg) [inline index.html:9]: could you please add some content here ?

File: index.html (hunk 1 above)
Comment (Tauqeer-Ahmed-99) [reply index.html:7]: Yes

File: index.html (hunk 2 above)
Comment (Naufil-Shemle) [reply index.html:9]: Yes sure

**Review by anask601 — APPROVED** [review]

**Review by faran4engg — APPROVED** [review]
looks good to me !

**Review by Rehmatulla-Khan — APPROVED** [review]

**Pull request merged**